src/presentation/http/router/note.test.ts

```diff
@@ -385,6 +385,58 @@ describe('Note API', () => {
     test.todo('Returns 400 when parentId has incorrect characters and lenght');
   });
 
+  describe('DELETE /note/:notePublicId', () => {
+    test('Remove note', async () => {
```

Comment:
do we need it along with https://github.com/codex-team/notes.api/pull/177?

Reply:
I think no, i will delete